src/Microsoft.AspNet.Mvc.Core/ActionContext.cs

```diff
@@ -12,6 +13,19 @@ namespace Microsoft.AspNet.Mvc
     /// </summary>
     public class ActionContext
     {
+        private ActionDescriptor _actionDescriptor;
+        private HttpContext _httpContext;
+        private ModelStateDictionary _modelState;
+        private RouteData _routData;
+
+        /// <summary>
+        /// Creates an empty <see cref="ActionContext"/>.
+        /// </summary>
+        public ActionContext()
+        {
+            // This default constructor should be used only for the controller unit test scenarios.
```

Comment:
This should go in the summary or remarks. It's not visible as part of the documentation otherwise.